src/index.css

```diff
@@ -0,0 +1,121 @@
+html {
+  margin:0;
+}
+body {
+  margin: 0 !important; 
+  height: 100vh;
+  width: 100vw;
+  background-color: #fffaea;
+}
+
+@font-face {
+  font-family: 'geeseband';
+  src: url('./assets/geeseband.otf');
+}
+
+@font-face {
+  font-family: 'geeseband';
+  src: url('./assets/Caslon Ionic-Regular Italic.otf');
+  font-style: italic;
+}
+
+.main_div {
```

Comment:
in reality to get this to work as expected I arranged all the main components inside this main_div that is a flex box. html and body are also set to full height without margins